src/Collection.php

```diff
@@ -810,6 +811,28 @@ public function listIndexes(array $options = [])
     }
 
     /**
+     * Runs map-reduce aggregation operations as defined by the $map and $reduce parameters.
```

Comment:
I'd suggest "Executes a map-reduce aggregation on the collection." based on `aggregate()`.